content/en/docs/examples/local-testing-with-kind/index.md

```diff
@@ -0,0 +1,391 @@
+---
+title: Testing Istio Locally With Kubernetes In Docker
+description: Example of deploying Istio locally using Kubernetes In Docker for testing and evaluation.
+weight: 10
+keywords:
+- kubernetes
+- kind
+- metallb
+- cert-manager
+- ingress
+test: true
+---
+
+## About
+
+This example uses Kubernetes In Docker ([KIND][kind]) to deploy Istio locally. The cluster produced is intended to be a supplement for further testing, evaluation and experimentation with Istio.
+
+## Before you begin
+
+Make sure that you have [Docker][docker] and [Kind][kindquickstart] installed and that you are able to create and use kind clusters and connect to them with [Kubectl][kubectl].
+
+## Deploying Kubernetes
```

Comment:
This and the previous section seems to duplicate material in the platform setup instructions for `kind`: https://preliminary.istio.io/docs/setup/platform-setup/kind/. Should it reuse, or at least explain how it relates to, those instructions?

Reply:
I've tried adding a reference below in a "tip" section that points to the document and recommends familiarizing yourself with it before proceeding. Let me know what you think.